src/main/java/touro/GameOfLife/GameOfLifeFrame.java

```diff
@@ -2,28 +2,38 @@
 
 import javax.swing.*;
 import java.awt.*;
-import java.io.IOException;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
 
 public class GameOfLifeFrame extends JFrame {
     public GameOfLifeFrame(Grid grid,
-                           GridView gridView,
-                           MouseClickEventListener mouseClick)
+                           GridView gridView)
         throws HeadlessException {
         super();
 
-        setSize(grid.WIDTH, grid.HEIGHT + 10);
+        setSize(500, 550);
```

Comment:
I was playing around with the numbers, and it looks like 515 x 563 fits the squares a little better